/camera_pose_estimator:
  ros__parameters:
    base_frame: 'world'
    camera_frame: 'camera_link'
    tag_frame_prefix: 'reftag_'
    publish_tags_tf: False
    tags_ids: [      1,       2,       3,       4,       5,       6,       7,       8,       9,      10,      11,      12]
    tags_x:   [-0.2875, -0.2875, -0.2875,  0.0000,  0.0000,  0.2875,  0.2875,  0.2875, -0.1438, -0.1438,  0.1438,  0.1438]
    tags_y:   [-0.2875,  0.0000,  0.2875, -0.2875,  0.2875, -0.2875,  0.0000,  0.2875, -0.1438,  0.1438, -0.1438,  0.1438]
